pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -319,8 +319,20 @@ func handleEvent(s *Server) {
 
 	// Only updating intermediate CA is supported now
```

Comment:
The PR was created a while ago, but now that root cert rotation is supported, can we remove this comment?